### PR TITLE
Make icds an alias to icds-new

### PR DESCRIPTION
**Pull request by dannyroberts**
Thoughts?

### CODE REVIEW

File: environments/icds [hunk 1]
```diff
@@ -0,0 +1 @@
+icds-new
```
Comment (dannyroberts) [inline environments/icds]: This is a symlink if that wasn't clear.

**Comment by snopoke** [comment]
@czue The folder structure is already just `icds` and not `icds-new`.

**Comment by czue** [comment]
oh great! ignore me.

**Comment by dannyroberts** [comment]
Yeah that's right. We've tried to make it so the folder name _only_ affects how you reference it in tools like ansible/fab/commcare-cloud, and does _not_ affect any of the setup. Folder names, supervisorctl task names, and such are determined by the (somewhat redundant) variables `environment` in `app-processes.yml` and `deploy_env` in `public.yml`.

**Comment by dannyroberts** [comment]
Oh and `env_name` in `public.yml`, which is meant to be unique, for use in things like datadog, email subjects, e.g. "icds-new". That's versus `deploy_env` and `environment` which are meant to be the "conceptual" instance, and can be shared between two environments that represent the same deployment (esp. during a migration), e.g. "icds".

Writing that down, it strikes me that that's super not clear from the variable names.

**Pull request merged**